objects/ocamera/Step_2.gml

```diff
@@ -1,13 +1,22 @@
 /// @description Insert description here
 // You can write your code in this editor
 
-if (p1 != noone && p2 != noone && p1.state != eState.HITSTOP && p2.state != eState.HITSTOP)
+// Set the camera's target
+if (p1 != noone && p2 != noone)
 {
-	x = (p1.x + p2.x) * .5;
+	// Camera's target is the midpoint between both players
+	xTo = (p1.xHome + p2.xHome) * .5;
 }
+
+// Move the camera
 if (room != rStageArcade)
 {
+	x = lerp(xTo, x, cameraSpeed);
+	
 	camera_set_view_pos(view_camera[0], clamp(x-(camWidth*.5), 0, camWidth), 0);

```

Comment:
```camWidth``` and ```camHeight``` should use the full word for camera, because there is no reason not to, the code style guide says not to abbreviate, and for consistency with other variable names, like ```cameraSpeed```